src/index.ts

```diff
@@ -19,9 +19,19 @@ interface Config {
 // TODO: we need to give a way to consumers to ensure this does not
 // run multiple times on a single page load.
```

Comment:
Lets remove this comment - as we decided this should be handled by the consumer lib.